src-java/reroute-topology/reroute-storm-topology/src/main/java/org/openkilda/wfm/topology/reroute/bolts/OperationQueueBolt.java

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2020 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.reroute.bolts;
+
+import static org.openkilda.wfm.topology.utils.KafkaRecordTranslator.FIELD_ID_KEY;
+import static org.openkilda.wfm.topology.utils.KafkaRecordTranslator.FIELD_ID_PAYLOAD;
+
+import org.openkilda.messaging.MessageData;
+import org.openkilda.messaging.command.CommandData;
+import org.openkilda.messaging.command.CommandMessage;
+import org.openkilda.messaging.command.flow.FlowPathSwapRequest;
+import org.openkilda.messaging.command.flow.FlowRerouteRequest;
+import org.openkilda.messaging.info.reroute.PathSwapResult;
+import org.openkilda.messaging.info.reroute.RerouteResultInfoData;
+import org.openkilda.wfm.CommandContext;
+import org.openkilda.wfm.error.PipelineException;
+import org.openkilda.wfm.share.hubandspoke.CoordinatedBolt;
+import org.openkilda.wfm.topology.reroute.service.OperationQueueService;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.storm.topology.OutputFieldsDeclarer;
+import org.apache.storm.tuple.Fields;
+import org.apache.storm.tuple.Tuple;
+import org.apache.storm.tuple.Values;
+
+@Slf4j
+public class OperationQueueBolt extends CoordinatedBolt implements OperationQueueCarrier {
+    public static final String BOLT_ID = "operation-queue";
+
+    public static final String FLOW_ID_FIELD = RerouteBolt.FLOW_ID_FIELD;
+
+    public static final String REROUTE_QUEUE_STREAM = "reroute-queue-stream";
+    public static final Fields REROUTE_QUEUE_FIELDS = new Fields(FLOW_ID_FIELD, FIELD_ID_PAYLOAD, FIELD_ID_CONTEXT);
+
+    public static final Fields FLOW_HS_FIELDS = new Fields(FIELD_ID_KEY, FIELD_ID_PAYLOAD);
+
+    private transient OperationQueueService service;
+
+    public OperationQueueBolt(int defaultTimeout) {
+        super(true, defaultTimeout);
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
... code is better without this garbage.

Reply:
Removed.